core/domain/opportunity_services.py

```diff
@@ -525,6 +526,28 @@ def delete_exp_opportunities_corresponding_to_story(story_id: str) -> None:
     exp_opprtunity_model_class.delete_multi(list(exp_opportunity_models))
 
 
+def get_model_for_delete_exp_opportunities(
```

Comment:
Same note as above. Don't pass storage models around in the domain layer, only pass domain objects around.